src/actions/README.md

```diff
@@ -8,14 +8,14 @@ All actions in the Firefox Profiler are fully typed using [Flow](https://flow.or
 
 # Preferred practices for actions
 
-Actions in the profiler are kept relatively simple. There are two types of action creators in this project–action creators that return an `Action` or an action creator that returns a `ThunkAction` (see [Redux Thunk](https://github.com/gaearon/redux-thunk) for additional reading.) If an action creator needs access to different parts of the state, then it may be tempting to use the `getState` parameter from a `ThunkAction`. However, at this time we prefer not to directly access the state in a `ThunkAction`. Instead, the connected component should look up the value using a selector, then pass it into the action creator as a parameter.
+Actions in the profiler are kept relatively simple. There are two types of action creators in these project–action creators that return an `Action` or an action creator that returns a `ThunkAction` (see [Redux Thunk](https://github.com/gaearon/redux-thunk) for additional reading.) If an action creator needs access to different parts of the state, we strongly recommend you use the `getState` parameter from a `ThunkAction`. If you need an alternative method to directly access the state in a `ThunkAction`, you can look up the value using a selector within the connected component. You can then pass it into the action creator as a parameter. 
```

Comment:
```suggestion
Actions in the profiler are kept relatively simple. There are two types of action creators in this project–action creators that return an `Action` or an action creator that returns a `ThunkAction` (see [Redux Thunk](https://github.com/gaearon/redux-thunk) for additional reading.) If an action creator needs access to different parts of the state, we strongly recommend you use the `getState` parameter from a `ThunkAction`. If you need an alternative method to directly access the state in a `ThunkAction`, you can look up the value using a selector within the connected component. You can then pass it into the action creator as a parameter. 
```

> There are two types of action creators in `this/these` project

Small tweak, I believe "this" refers to project, which is singular.

Reply:
I can't add a commit to your branch @lloan , so if you merge this suggestion I can merge in the work.

Reply:
Changed it - thanks!